app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -9,16 +9,22 @@ class MS.Views.LoanQuestionsView extends Backbone.View
       data: @tree.data('data')
       dragAndDrop: true
       selectable: false
+      onCanMove: (node) =>
+        if node.can_edit == true
+          return true
+        else
+          return false
```

Comment:
I think this could be written as `onCanMove: (node) => node.can_edit`. If it's not clear why, I'm happy to quickly discuss it.